app/src/main/kotlin/edu/stanford/bdh/engagehf/education/EngageEducationRepository.kt

```diff
@@ -0,0 +1,20 @@
+package edu.stanford.bdh.engagehf.education
+
+import com.google.firebase.firestore.FirebaseFirestore
+import edu.stanford.spezi.module.onboarding.invitation.await
+import edu.stanford.spezi.modules.education.videos.VideoSection
+import edu.stanford.spezi.modules.education.videos.data.repository.EducationRepository
+import javax.inject.Inject
+
+class EngageEducationRepository @Inject constructor(
+    private val firebaseFirestore: FirebaseFirestore,
```

Comment:
the fact that only firebase specific things are used here is engage specific. in a final state, the account module would have to provide an interface, which can then be provided via a firebase module default, but alternatively e.g. AWS can also be used instead of firebase. at first i thought that since we use so many firebase specific, there will be few providers to replace it with, but [supabase](https://supabase.com/) would be a conceivable option, for example